README.md

```diff
@@ -1,2 +1,25 @@
-# pidgin
-Light weighted restful api on top of peregrine
+# Pidgin
```

Comment:
this may be out of the scope of these docs, but it may make sense to handle the case where peregrine isn't running more gracefully. or at least make sure to note somewhere in these docs that you cannot run this service alone. when I try to `python run.py` I get an error: `Max retries exceeded with url: /v0/submission/graphql/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2efa627f90>: Failed to establish a new connection: [Errno 111] Connection refused',))
`

Reply:
that's if I hit the root. but if I hit the _status I get Healthy so idk

Reply:
I created a Jira user story for that